README.md

```diff
@@ -1,2 +1,55 @@
 # gradle-crowdin-cli-plugin
 A gradle plugin to utilize the crowdin cli to interact with crowdin
+
+## Usage
+
+You need to copy the crowin-cli into you project and configure where to find the executable cli jar file (relative to your build file):
+
+```
+crowdin {
+  cli = 'crowdin-cli.jar'
+}
+```
+
+The plugin adds two task which use the default crowdin config file `crowdin.yaml`. You can upload source files via `crowdinUpload` and download translated files via `crowdinDownload`.
+
+## Custom Tasks
+
+You can create own task to customize both upload and download depending on you configuration:
+
+```
+task uploadCustom (type: CrowdinUpload) {
+  config = 'myCustomConfig.yaml'
```

Comment:
Wouldn't it be cool, to have the crowdin config as a gradle extension? 
No addtional files needed.

Reply:
yes, but for us using the config file is more easier (now), as we have everything ready and we don't need to change the plugin when the config format changes. A plugin wrapping the configuration into gradle is this for example: https://github.com/mendhak/gradle-crowdin-plugin

In the long term we might support both, but I think as long as we utilize the cli (and not the api directly) going with config file is the better solution.